examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -103,7 +118,43 @@ async function example_API_Delete(cacheClient: CacheClient) {
   }
 }
 
+function example_API_InstantiateAuthClient() {
+  new AuthClient({
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_AUTH_TOKEN',
+    }),
+  });
+}
+
+async function example_API_GenerateAuthToken(authClient: AuthClient) {
+  const generateTokenResponse = await authClient.generateAuthToken(AllDataReadWrite, ExpiresIn.minutes(30));
+  if (generateTokenResponse instanceof GenerateAuthToken.Success) {
+    console.log('Auth token generated!');
+    console.log(`Auth token starts with: ${generateTokenResponse.authToken.substring(0, 10)}`);
+    console.log(`Refresh token starts with: ${generateTokenResponse.refreshToken.substring(0, 10)}`);
```

Comment:
Initially I was confused as to why we were just logging what the token starts with, but I'm guessing it's because we don't want to print the whole thing for security? I guess I'm slightly worried that people are going to thing the `substring(0,10)` has some kind of meaning?

Reply:
https://github.com/momentohq/client-sdk-javascript/pull/542/commits/dc0615785b66c8d18d2e4d3e217d8c42efb71353